login-fire-common-behavior.html

```diff
@@ -291,6 +291,17 @@
       resources: {
         type: Object,
         readOnly: true
+      },
+
+      /**
+       * The _inProgress indicator shows whether a sign in or sign out process is currently running.
```

Comment:
As recommended in the [Polymer documentation](https://www.polymer-project.org/2.0/docs/tools/documentation#language), try to follow the [JavaDoc style](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide). I would have wrote something like: _"Indicates if a sign in or sign out process is in progress."_

Reply:
Agree. Will change it.